metrics/influxdb/influxdb.go

```diff
@@ -0,0 +1,78 @@
+// Package influxdb implements a Influxdb backend for package metrics.
+package influxdb
+
+import (
+	"time"
+
+	"github.com/go-kit/kit/metrics"
+	stdinflux "github.com/influxdata/influxdb/client/v2"
+)
+
+type influxdbCounter struct {
```

Comment:
You can just call this e.g. `counter`, influxdb is implicit from the package name.